### ISSUE TITLE
Unable to record when trying to record audio get unknown error

### ISSUE THREAD

**Issue by justinmiller62** [BradLarson/GPUImage2#139]
Here is the error code, I get the same thing whether using my app or the sample app. 

Optional(Error Domain=AVFoundationErrorDomain Code=-11800 "The operation could not be completed" UserInfo={NSUnderlyingError=0x618000058270 {Error Domain=NSOSStatusErrorDomain Code=-12780 "(null)"}, NSLocalizedFailureReason=An unknown error occurred (-12780), NSLocalizedDescription=The operation could not be completed})
Trouble appending audio sample buffer

**Comment by jmm55** [comment]
It looks like I was able to fix this by modifying the "MovieOutput.swift" file, I confirm that I can now record video and synced audio on the mac version of the framework, I have not tested to see if there is an issue on the iOS side or if this will fix that.  I am attaching the file, I will see if I can do a pull request and commit it for your approval. 
[MovieOutput.swift.zip](https://github.com/BradLarson/GPUImage2/files/885623/MovieOutput.swift.zip)

**Comment by justinmiller62** [comment]
It looks like I was able to fix this by modifying the "MovieOutput.swift" file, I confirm that I can now record video and synced audio on the mac version of the framework, I have not tested to see if there is an issue on the iOS side or if this will fix that. I am attaching the file, I will see if I can do a pull request and commit it for your approval.

[MovieOutput.swift.zip](https://github.com/BradLarson/GPUImage2/files/885640/MovieOutput.swift.zip)

**Comment by henvyluk** [comment]
Is your problem solved? I also encountered such a problem, hope some help.@justinmiller62

**Comment by justinmiller62** [comment]
yes the attached zip file has the fix in it. I did do a merge request but I am uncertain whether that was added to the project.

**Comment by henvyluk** [comment]
it was also a Mac version,  but the iOS version I was trying to be unsuccessful@justinmiller62
